TestServer/Unit/BookingServiceTests.cs

```diff
@@ -0,0 +1,106 @@
+using Moq;
+using server.Models.Domain;
+using server.Models.DTOs.Internal;
+using server.Models.DTOs.Request;
+using server.Repository;
+
+namespace server.Services.Internal.Tests
+{
+    public class BookingServiceTests
+    {
+        private readonly Mock<IBookingRepository> _bookingRepositoryMock = new Mock<IBookingRepository>();
```

Comment:
Using Moq.AutoMock, you can avoid manual setup of mock objects mock object remains consistent throughout the lifetime of the test class.